sakai-evaluation-tool/src/java/org/sakaiproject/evaluation/tool/producers/ReportsViewingProducer.java

```diff
@@ -589,18 +628,22 @@ private void renderReportingOptionsTopLinks(UIContainer tofill, EvalEvaluation e
 				  EvalUser user = commonLogic.getEvalUserById( evaluatee.getUserId() );
 				
 				  UIInternalLink.make(evaluateeBranch, "pdfResultsReportIndividualLink", UIMessage.make("viewreport.view.pdf.individual", new Object[] {user.displayName}), new DownloadReportViewParams(
-						  EvalEvaluationService.PDF_RESULTS_REPORT_INDIVIDUAL, templateId, reportViewParams.evaluationId, reportViewParams.groupIds, evaltitle+"Individual.pdf", evaluatee.getUserId()));
+						  EvalEvaluationService.PDF_RESULTS_REPORT_INDIVIDUAL, templateId, reportViewParams.evaluationId, reportViewParams.groupIds, evaltitle+"Individual.pdf", evaluatee.getUserId(), false));
                   listedEvaluatees.add(evaluatee.getUserId());
                 }
 			}
 			
         }
 
         // FIXME should this be protected with an option?
+        // New report style - csv eval takers export button and parameters
         Boolean allowListOfEvalTakers = (Boolean) evalSettings.get(EvalSettings.ENABLE_LIST_OF_TAKERS_EXPORT);
-        if (allowListOfEvalTakers != null && allowListOfEvalTakers == true) {
-            UIInternalLink.make(tofill, "exportListOfEvaluationTakers", UIMessage.make("viewreport.view.listofevaluationtakers"), new DownloadReportViewParams(
-                    "csvTakersReport", templateId, reportViewParams.evaluationId, reportViewParams.groupIds, evaltitle+"-takers.csv"));
+        if( allowListOfEvalTakers != null && allowListOfEvalTakers == true )
```

Comment:
This has already been replaced with the following:

`if( Boolean.TRUE.equals( allowListOfEvalTakers ) )`